wget/README.md

```diff
@@ -0,0 +1,51 @@
+
+# `wget`
+
+This task performs wget operation to download file from internet to workspace .
```

Comment:
```suggestion
This task uses wget  to download files from the internet to a workspace .
```

Reply:
Thanks, Dibyo, now the description makes more sense!